### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by saharjs**
Hi
Required by the one of the assignments of my technical writing course I am supposed to edit other people's writing. I think the minor changes that I have proposed would increase the readability of your README file.
Thank you

### CODE REVIEW

**Comment by vtjnash** [comment]
Welcome and thanks for taking an interest in Julia! It's great that you took a school assignment and turned it into an opportunity to dive into open source. 🙂

File: README.md [hunk 1]
```diff
@@ -68,16 +68,16 @@ By default you will be building the latest unstable version of Julia. However, m
 
 Now run `make` to build the `julia` executable. To perform a parallel build, use `make -j N` and supply the maximum number of concurrent processes. (See [Platform Specific Build Notes](https://github.com/JuliaLang/julia#platform-specific-build-notes) for details.)
 When compiled the first time, it will automatically download and build its [external dependencies](#required-build-tools-and-external-libraries).
-This takes a while, but only has to be done once. If the defaults in the build do not work for you, and you need to set specific make parameters, you can save them in `Make.user`. The build will automatically check for the existence of `Make.user` and use it if it exists.
+This takes a while, but only has to be done once. If the defaults in the build do not work for you, and you need to set specific make parameters which can be saved in `Make.user`. The build will automatically check for the existence of `Make.user` and use it if it exists.
```
Comment (vtjnash) [inline README.md]: The first part of this sentence is still in the active voice, so the second part should parallel that rather than switching to the passive voice. You could rewrite the whole sentence, but I think this is probably more easily readable in the active voice.

File: README.md [hunk 2]
```diff
@@ -68,16 +68,16 @@ By default you will be building the latest unstable version of Julia. However, m
...
 Building Julia requires 1.5GiB of disk space and approximately 700MiB of virtual memory.
 
 For builds of julia starting with 0.5.0-dev, you can create out-of-tree builds of Julia by specifying `make O=<build-directory> configure` on the command line. This will create a directory mirror, with all of the necessary Makefiles to build Julia, in the specified directory. These builds will share the source files in Julia and `deps/srccache`. Each out-of-tree build directory can have its own `Make.user` file to override the global `Make.user` file in the top-level folder.
 
-If you need to build Julia in an environment that does not allow access to the outside world, use `make -C deps getall` to download all the necessary files. Then, copy the `julia` directory over to the target environment and build with `make`.
+If you need to build Julia in an isolated environment, use `make -C deps getall` to download all the necessary files. Then, copy the `julia` directory over to the target environment and build with `make`.
```
Comment (vtjnash) [inline README.md]: I agree this was too wordy, so this change is good. But perhaps "...on a machine without internet access" would be even more clear?

File: README.md [hunk 3]
```diff
@@ -68,16 +68,16 @@ By default you will be building the latest unstable version of Julia. However, m
...
 
-**Note:** the build process fail badly if any of the build directory's parent directories have spaces or other shell meta-characters such as `$` or `:` in their names (this is due to a limitation in GNU make).
+**Note:** the build process fails badly if any of the build directory's parent directories have spaces or other shell meta-characters such as `$` or `:` in their names (this is due to a limitation in GNU make).
 
-Once it is built, you can run the `julia` executable using its full path in the directory created above (the `julia` directory), or, to run it from anywhere, either
+Once it is built, you can run the `julia` executable using its full path in the directory created above (the `julia` directory). In addition you can run it from anywhere using either following way:
```
Comment (vtjnash) [inline README.md]: There needs to be a preposition between "either" and "following way" (i.e. "either of the following ways"). There should also be a comma after the conjunctive adverb "In addition".

File: README.md [hunk 4]
```diff
@@ -92,13 +92,11 @@ Now you should be able to run Julia like this:
...
-If everything works correctly, you will see a Julia banner and an interactive prompt into which you can enter expressions for evaluation. (Errors related to libraries might be caused by old, incompatible libraries sitting around in your PATH. In that case, try moving the `julia` directory earlier in the PATH).
+If everything works correctly, you will see a Julia banner and an interactive prompt into which you can enter expressions for evaluation. (Errors related to libraries might be caused by old, incompatible libraries sitting around in your PATH. In this case, try moving the `julia` directory earlier in the PATH).
 
-Your first test of Julia should be to determine whether your
-build is working properly. From the UNIX/Windows command prompt inside
+Your first test of Julia determines whether your build is working properly. From the UNIX/Windows command prompt inside
 the `julia` source directory, type `make testall`. You should see output
-that lists a series of tests being run; if they complete without
-error, you should be in good shape to start using Julia.
+that lists a series of running tests; if they complete without error Julia can be started.
```
Comment (vtjnash) [inline README.md]: The verb here was "use", so the direct translation into the passive voice would be "the use of Julia can start". That's rather awkward though, since the use of passive voice is not required in the readme. Although, the current wording is quite informal, which might not be ideal either. Maybe you could rewrite this to be more precise, but keep it in the active voice?

Optionally, another useful addition here might be to mention "If something isn't working right, please file a [bug report](https://github.com/JuliaLang/julia/issues/new)! That may be the only way that the developers find out that there is a bug and who it affects." (ok, that's not my favorite sentence structure either, maybe you could help me with word-smithing it also 🙂 )

File: README.md [hunk 5]
```diff
@@ -68,16 +68,16 @@ By default you will be building the latest unstable version of Julia. However, m
...
 
-If you need to build Julia in an environment that does not allow access to the outside world, use `make -C deps getall` to download all the necessary files. Then, copy the `julia` directory over to the target environment and build with `make`.
+If you need to build Julia in an isolated environment, use `make -C deps getall` to download all the necessary files. Then, copy the `julia` directory over to the target environment and build with `make`.
 
-**Note:** the build process fail badly if any of the build directory's parent directories have spaces or other shell meta-characters such as `$` or `:` in their names (this is due to a limitation in GNU make).
+**Note:** the build process fails badly if any of the build directory's parent directories have spaces or other shell meta-characters such as `$` or `:` in their names (this is due to a limitation in GNU make).
```
Comment (apurvis) [inline README.md]: looks slightly better to capitalize `The build process...`
